UI/src/less/site/pages.less

```diff
@@ -129,3 +145,13 @@
     background-color: #dddddd;
   }
 }
+
+.bigListPopUp {
+  overflow-y: auto;
+  height: auto;
+  max-height: 500px;
+}
+
+.clickableText {
+  cursor: pointer;
+}
```

Comment:
You already have this one you can use:

.selectable{
  cursor: pointer;
}

Reply:
clickableText removed and selectable applied